src/rulesets/LinearAlgebra/dense.jl

```diff
@@ -194,10 +194,21 @@ function frule((_, ΔA), ::typeof(pinv), A::AbstractMatrix{T}; kwargs...) where
     return Y, ∂Y
 end
 
+function rrule(
+    ::typeof(pinv),
+    x::Union{AbstractVector{T}, LinearAlgebra.AdjOrTransAbsVec{T}},
+) where {T<:Union{Real,Complex}}
+    y, full_pb = rrule(pinv, x, 0)
+    function pinv_pullback(Δy)
+        return full_pb(Δy)[1:2]
+    end
```

Comment:
I am trying to remeber if there is a reason we don't do this:
```suggestion
    pinv_pullback(Δy) = full_pb(Δy)[1:2]
```